packages/env-loader/src/cli-options.ts

```diff
@@ -66,8 +66,6 @@ export const getCliOptionsAndArgs = (): CliOptionsAndArgs => {
 
   program
     .passThroughOptions()
-    .option('--DEBUG', 'Verbose logging on build', false)
-    .option('--no-DEBUG', 'No verbose logging on build')
```

Comment:
This was used really only to show all `proxy-fetcher` failures, but to do that, it was overriding the `DEBUG` environment variable, which [`debug`](https://www.npmjs.com/package/debug) uses.